src/main/java/bibi/Storage.java

```diff
@@ -0,0 +1,91 @@
+package bibi;
+
+import bibi.task.*;
```

Comment:
Should this wildcard import be removed as it does not follow the coding standard?